examples/src/erc20.rs

```diff
@@ -12,7 +12,7 @@ pub struct Erc20 {
     name: Variable<String>,
     total_supply: Variable<U256>,
     balances: Mapping<Address, U256>,
-    allowances: Mapping<(Address, Address), U256>
+    allowances: Mapping<Address, Mapping<Address, U256>>
```

Comment:
I'd keep the old style. It's probably faster. Noo need to to create intermediate Mapping here. Setters and getters are more readable.